listenbrainz/mbid_mapping/docker/mapper.service

```diff
@@ -1,3 +1,3 @@
 #!/bin/bash
```

Comment:
The permissions for this file seem to have changed. Just to confirm, that's intended or harmless?

Reply:
Intended.